components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -998,23 +998,29 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
                 prebuiltWorkspace = await this.workspaceDb
                     .trace(ctx)
                     .findPrebuiltWorkspaceByCommit(cloneUrl, commitSHAs);
+                if (prebuiltWorkspace?.state !== "available" && allowUsingPreviousPrebuilds) {
```

Comment:
This slight change in behavior (never starting based on previous prebuilds for OpenPrebuildContext) makes sense :100:

Reply:
Would it make sense to change this to `prebuiltWorkspace?.state !== "available" && ignoreRunningPrebuild && allowUsingPreviousPrebuilds`?
Otherwise exposing the `ignoreRunningPrebuild` option to users makes less sense, because it is not independent of `allowUsingPreviousPrebuild`, and we would have to explain that connection. :confused: